wrappers/javascript/indy-vdr-nodejs/package.json

```diff
@@ -40,10 +40,11 @@
     "typescript": "~4.9.4"
   },
   "dependencies": {
-    "@hyperledger/indy-vdr-shared": "0.2.0-dev.6",
-    "@mapbox/node-pre-gyp": "^1.0.10",
     "@2060.io/ffi-napi": "4.0.8",
     "@2060.io/ref-napi": "3.0.6",
+    "@hyperledger/indy-vdr-shared": "0.2.0-dev.6",
+    "@mapbox/node-pre-gyp": "^1.0.10",
+    "@types/ref-array-di": "^1.2.8",
```

Comment:
it was about the `@types/ref-array-di`, `@mapbox/node-pre-gyp` or both?

For the first one I think I'll probably need to add it to `@2060.io/ffi-napi` dependencies

Reply:
It's for `@types/ref-array-di`, the other ones are just reordered in the list. Adding them to `@2060.io/ffi-napi` might work, but I think the error was coming from `indy-vdr`